app/apps/cases/mock.py

```diff
@@ -0,0 +1,131 @@
+import datetime
```

Comment:
De views.py voor cases werd steeds groter vanwege deze mock, dus ik heb dit verplaatst naar een dedicated mock file.

Reply:
dan heb ik liever een views/* constructie. Maar maakt niet uit want deze mock cases heeft niet lang voor leven door camunda

Reply:
De views wordt wel erg groot en overzichtelijk dan, maar als dit er toch uit moet laten we het ff zo.